### PR TITLE
Change adding and deleting students from lessons feature to accept only one student and one lesson at a time

### PR DESCRIPTION
**Pull request by ErnestCuong**
Changes the feature to accept only one student into one lesson at a time

### CODE REVIEW

File: src/main/java/tutoraid/model/lesson/Students.java [hunk 1]
```diff
@@ -57,10 +57,10 @@ public void addStudent(Student student) {
      */
     public void removeStudent(Student student) {
         requireNonNull(student);
+        System.out.println(students.contains(student));
```
Comment (wpinrui) [inline src/main/java/tutoraid/model/lesson/Students.java:60]: Was this intentionally left in or maybe you used for debugging?

File: src/main/java/tutoraid/model/student/Lessons.java [hunk 2]
```diff
@@ -7,8 +7,8 @@
 import tutoraid.commons.util.AppUtil;
```
Comment (wpinrui) [inline src/main/java/tutoraid/model/student/Lessons.java:7]: Is this the class that we might be changing to LessonNames?

**Review by wpinrui — APPROVED** [review]
LGTM

File: src/main/java/tutoraid/model/lesson/Students.java (hunk 1 above)
Comment (ErnestCuong) [reply src/main/java/tutoraid/model/lesson/Students.java:60]: Ah thank you for notifying me this, I was debugging. Will delete it now.

File: src/main/java/tutoraid/model/student/Lessons.java (hunk 2 above)
Comment (ErnestCuong) [reply src/main/java/tutoraid/model/student/Lessons.java]: Yes

**Review by dt-td — APPROVED** [review]
LGTM

**Pull request merged**